lte/gateway/docker/services/python/Dockerfile

```diff
@@ -43,6 +43,7 @@ RUN apt-get update && apt-get install -y \
   sudo \
   ruby-dev \
   docker.io \
+  python-is-python3 \
```

Comment:
do we actually need this in the containerized agw? The invocations I see so far mention python3 explicitly.

Reply:
We probably don't need it actually. From what I saw, Python 2 isn't installed and there is no `python` symlink without a typo at the moment.

I was trying to preserve the intended behavior but I could also remove it, what do you think?

Reply:
I only looked at the "dev/production" image, not the "builder" image though.

Reply:
I don't have a strong opinion, but I'm slightly leaning towards not having a `python` binary and encouraging people to be explicit when invoking python. That is, as long as that's feasible. In this case I guess we don't need the symlink given that the old symlink was broken anyway and the images still seemed to work.

Reply:
Done.